ParisBNF/abb/BNFabb20.xml

```diff
@@ -39,11 +39,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         </msItem>
```

Comment:
Maybe since we are at this file we can also correct the spelling in line 7 ('Afa Warq)

Reply:
I suggest to change the title to the title of the main work, that is: Tǝrgʷāme malʾǝkt za-bǝṣuʿ wa-kǝbur Ṗāwlos za-ṣaḥafa la-ʿƎbrāwǝyān.

Reply:
Would you suggest to keep it like this or should I restore the name ʾAfa Warq as given in the catalogue?

Reply:
I do not find this important. Whatever. I like to make the records more accessible to general reader so I probably would have written "Commentary on the Hebrews" but it is really not important.

Reply:
The catalogue reproduces the label on the binding
![image](https://user-images.githubusercontent.com/17987728/219346858-3d119906-3950-43b9-9007-ca9ca35ec002.png)